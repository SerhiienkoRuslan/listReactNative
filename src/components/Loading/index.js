import React from 'react';
import { View, ActivityIndicator } from 'react-native';

import colors from 'styles/colors';

import styles from './styles';

export default () => (
  <View style={styles.centered}>
    <ActivityIndicator size="large" color={colors.primary} />
  </View>
);
