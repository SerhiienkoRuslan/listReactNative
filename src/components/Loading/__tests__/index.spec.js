import Loading from '../index';

describe('Loading component', () => {
  it('should render Loading component', () => {
    const component = shallow(<Loading />);
    expect(component).toMatchSnapshot();
  });
});
