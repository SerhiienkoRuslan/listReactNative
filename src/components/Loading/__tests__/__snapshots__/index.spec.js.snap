// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render Loading component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#fb5b5a"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
